src/Day1.hs

```diff
@@ -65,7 +65,7 @@ classic :: Classic State
 classic = Classic
   { title = "Day 1"
   , rows = screenH
-  , cols = screenW
+  , cols = screenW + 1
```

Comment:
This defeats the purpose of `screenW`. Instead we should create another value named something like `roomWidth`. Such that `screenW = roomWidth + hudWidth`